### ISSUE TITLE
Shields look a little tight 

### ISSUE THREAD

**Issue by bsudekum** [mapbox/mapbox-navigation-ios#1227]
![57712988-7a7d-47e0-95be-771ffb111d13](https://user-images.githubusercontent.com/1058624/37465602-cf253be6-2818-11e8-84d1-2f342acd1446.png)

I believe this occurred in the abbreviations work, but the shields look a little tight. Perhaps we can add a little more margin right? 

Cc @frederoni

**Comment by frederoni** [comment]
Yep, definitely. Do we want a delimiter here too? ("US-101 / I-80")

**Comment by bsudekum** [comment]
I know we have gone back and forth on this, but my current thinking is any raw text the server gives us, we should display. If we want to remove delimiters, let’s do it on server. If adding back delimiters causes the ui to look ugly, we can dial this back slowly.

**Comment by 1ec5** [comment]
We should hide the delimiters as soon as the shields load. The reason the server includes the delimiter is that the shields may not be visible immediately; we wouldn’t want to have two numbers from different components separated only by a space.
